Cargo.lock

```diff
@@ -91,7 +91,7 @@ dependencies = [
 
 [[package]]
 name = "create-rust-app"
-version = "2.1.0"
+version = "2.1.1"
```

Comment:
```suggestion
version = "2.1.1"
```

was this a mistake? also, same thing with `package-lock.json`? (I don't see any `package.json` changes)

Reply:
cool